actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -252,6 +252,10 @@ def require(key)
     # Alias of #require.
     alias :required :require
 
+    def require_all(keys)
```

Comment:
Validations work well if you want to store a value. However, if you want to do some pre-processing on param before storing it, you have to add `before_validation` action which works smaller apps. But incase of larger apps where same model is being used in multiple controllers(Web vs API controllers for example), it gets you into conditional callbacks which are not exactly clean. 

This is where ability to guard against `Nil` at controller level  is simple, clean way where what params are needed are documented in controller itself. 

Now, I am curious to know what are the cons of this [approach](http://stackoverflow.com/questions/22487878/strong-parameters-require-multiple) according to you.


if you think this idea is good, then I will improve the code & try to address issues you have mentioned.